Remove or update references to NOSE customization

AdamCatalyst opened the pull request:
- Remove or update references to NOSE customization
- updated NOSE update.html to HTML5
- readme and PY version & year updates

AdamCatalyst committed:
Remove or update references to NOSE customization

- Remove or update references to NOSE customization
- updated NOSE update.html to HTML5
- readme and PY version & year updates

smozoma commented on smozROM.py:
```diff
@@ -38,37 +38,7 @@ def debug(str):
-    else:
-        os.system('clear')
-
-def wait_key():
```
These functions need to be restored or it won't work (clearScreen and wait_key)

AdamCatalyst replied:
addressed in new pull request.

smozoma commented on NOSE update.html:
```diff
@@ -1,97 +1,73 @@
-  <li>Paste these files into the '<span style="font-weight: bold;">ini</span>'
-directory
-of
+<body style="margin: 0 auto; padding: 2em 2.5em; max-width:808px;">
```
Can yo do me a favour and put "border: 1px solid;" back in? it looks weird without a border.

AdamCatalyst replied:
yer border is back. adjusted padding to match.

AdamCatalyst committed:
Comments Rd #1

- restored border (adjusted CSS for readable margins accordingly)
- restored code i accidentally deleted.
- updated ignore to ignore ds_store

AdamCatalyst committed:
Cleanup

smozoma reviewed (approved):
thanks!
